calc/calc.py

```diff
@@ -112,7 +112,7 @@ def lower_dict(input_dict):
 
 def super_float(text):
     """
-    Like float, but with SI extensions. 1k goes to 1000.
+    Evaluate suffixes if applicable and apply float.
```

Comment:
I would like to still see `SI extensions` mentioned and some example would also be helpful in the comment.